socket.c

```diff
@@ -78,6 +79,13 @@ bool cSatipSocket::Open(const int portP, const bool reuseP)
      ERROR_IF_FUNC(setsockopt(socketDescM, SOL_IP, IP_PKTINFO, &yes, sizeof(yes)) < 0,
                    "setsockopt(IP_PKTINFO)", Close(), return false);
 #endif // __FreeBSD__
+
+     if (rcvBufSizeM > 0)
+     {
+       ERROR_IF_FUNC(setsockopt(socketDescM, SOL_SOCKET, SO_RCVBUF, &rcvBufSizeM, sizeof(rcvBufSizeM)) < 0,
```

Comment:
Is this working in FreeBSD as well?

Reply:
According to the man page here, yes:
https://www.freebsd.org/cgi/man.cgi?query=setsockopt&sektion=2
But I have no way of testing it.